### ISSUE TITLE
[Feature Request] Move `refresh()` from `RootApi` to `FolderApi` ?

### ISSUE THREAD

**Issue by scarletsky** [cocopon/tweakpane#510]
Now we can only call `refresh()` on the RootApi,  but not on FolderApi.

```js
const pane = new Pane({ title: 'Pane' });
const folder = pane.addFolder({ title: 'folder' });

folder.refresh(); // not work, no `refresh` method on FolderApi.
pane.refresh(); // work
```

This will refresh all bindings.
If the pane has many fields, it would cause performance issue.

And your `RootApi#refresh` [implementation](https://github.com/cocopon/tweakpane/blob/c69ddc543845edc72f1f20a9ca4c8baf70218417/packages/tweakpane/src/main/ts/blade/root/api/root.ts#L53-L67) can safely move to `FolderApi`.

Now I am doing the same thing on my codebase.

```js
import('tweakpane').then(module => {
    const { Pane, FolderApi } = module;

    if (!FolderApi.prototype.refresh) {
        FolderApi.prototype.refresh = Pane.prototype.refresh;
    }
});
```

I think this will be useful.

**Comment by cocopon** [comment]
Implemented in #513 and it will be released in the next major version 4. Please wait for the next release...

**Comment by scarletsky** [comment]
Great ! Wait for v4. 😄
